sdk/core/core-rest-pipeline/test/browser/fetchHttpClient.spec.ts

```diff
@@ -39,6 +39,32 @@ function createResponse(statusCode: number, body = "", chunkDelay = 0): Response
   return new Response(stream, { status: statusCode });
 }
 
+function createStreamResponseWithAbortSignal(
```

Comment:
Could we use this helper everywhere instead of having two helpers where one is more general than the other?